chart/dapr-ambient/templates/_daemondeployshared.yaml

```diff
@@ -18,47 +18,19 @@
       serviceAccountName: {{ include "dapr-ambient.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
+        - name: setup-trust-bundle
+          image: "{{ .Values.ambient.proxy.image.registry }}/{{ .Values.ambient.proxy.image.name }}:{{ .Values.ambient.proxy.image.tag }}"
```

Comment:
@mcruzdev we need to change the name of the variable, as this is not a proxy anymore.. we can call it just `initcontainer` -> `.Values.ambient.initcontainer.image.registry`